test/fixtures/refract/path-level-params.json

```diff
@@ -115,13 +115,14 @@
                 }
               ]
             },
-            "href": "/test/{id}{?search,arg}"
+            "href": "/test/{id}{?search}"
           },
           "content": [
             {
               "element": "transition",
               "meta": {},
               "attributes": {
+                "href": "/test/{id}{?search,arg}",
```

Comment:
Looks like you cover the situation here I mentioned above, where a transition adds to the query parameters.